docs/resources/ec_deployment.md

```diff
@@ -97,98 +99,98 @@ resource "ec_deployment" "ccs" {
 
 The following arguments are supported:
 
-* `region` - (Required) ESS region where to create the deployment. For ECE environments "ece-region" must be set.
+* `region` - (Required) Elasticsearch Service (ESS) region where to create the deployment. For Elastic Cloud Enterprise (ECE), set the `"ece-region"` environment.
```

Comment:
the environment word seems disconnected here?

Reply:
Reworded in:

> Elasticsearch Service (ESS) region where to create the deployment. For Elastic Cloud Enterprise (ECE) installations, set `"ece-region"`.